lib/clowder-common-ruby/types.rb

```diff
@@ -221,6 +221,7 @@ def valid_keys
       [].tap do |keys|
         keys << :accessKey
         keys << :secretKey
+        keys << :region
```

Comment:
@psav Like the 1st comment line says: this file is automated created by run the script `sync_config.sh`, which will update types with the latest changes based on remote schema changes.